h2o-core/src/main/java/water/rapids/RadixCount.java

```diff
@@ -62,8 +64,8 @@ static Key getKey(boolean isLeft, int col, H2ONode node) {
           if (chk.isNA(r)) tmp[0]++;
           else {
             long ctrVal = isIntVal ?
-                    BigInteger.valueOf(chk.at8(r)).subtract(_base).add(BigInteger.ONE).shiftRight(_shift).longValue():
-                    MathUtils.convertDouble2BigInteger(chk.atd(r)).subtract(_base).add(BigInteger.ONE).shiftRight(_shift).longValue();
+                    BigInteger.valueOf(_ascending?chk.at8(r):chk.at8(r)*(-1)).subtract(_base).add(BigInteger.ONE).shiftRight(_shift).longValue():
```

Comment:
It would be a little bit more elegant if _ascending was an integer array instead (-1, +1) - then you can always multiply and without the ternary operator (it might be called "direction" instead of ascending).

Reply:
Good idea.  I am taking it.